File: Pessoas-Angular/src/app/modelos/pessoas-fones.ts
import { Pessoas  } from '../modelos/pessoas';
import { FoneTipo } from '../modelos/fone-tipo';

export class PessoasFones{
  id: number;
  foneTipoId: number;
  pessoasId: number;
  foneNumero: number;
  pessoas: Pessoas;
  foneTipo: FoneTipo;
}

File: PessoasFone.Modelos/Modelos/PessoasFones.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PessoasFone.Modelos.Modelos
{
    public class PessoasFones
    {
        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public int FoneNumero { get; set; }
        public int? PessoasId { get; set; }
        public int FoneTipoId { get; set; }
        public Pessoas Pessoas { get; set; }
        public FoneTipo FoneTipo { get; set; }
    }
}
